### PR TITLE
Digest #13

### PR DESCRIPTION
**Pull request by pjazdzewski1990**

### CODE REVIEW

File: _posts/9-01-2015-digest13.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- Scala
+---
+
+Hi folks! 
+
+{DESC}
+
+* \[Scala\] [Abstract types in Scala](https://www.hakkalabs.co/articles/scala-bay-tech-talk-abstract-types#)
+
+* \[Scala\] [Scala puzzlers](http://scalapuzzlers.com/)
```
Comment (marekklis) [inline _posts/9-01-2015-digest13.md]: :+1:

File: _posts/9-01-2015-digest13.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+* \[Javascript\] [Token-Based authentication for the web](http://code.tutsplus.com/tutorials/token-based-authentication-with-angularjs-nodejs--cms-22543)
+ 
+* \[Javascript\] [Reactive programming in Javascript at Netflix](http://www.infoq.com/presentations/async-programming-netflix)
+
+* \[Frontend\] [On the road to 2015](https://medium.com/@addyosmani/javascript-application-architecture-on-the-road-to-2015-d8125811101b)
+ 
+* \[Game Development\] [From Never Doing Game Code or Game Art... To A Full Game](http://www.gamedev.net/page/resources/_/business/breaking-into-the-industry/from-never-doing-game-code-or-game-art-to-a-full-game-in-unreal-engine-4-r3937)
+
+* \[Working\] [Devs should write](http://www.shubhro.com/2014/12/27/software-engineers-should-write/)
```
Comment (marekklis) [inline _posts/9-01-2015-digest13.md]: :+1:

File: _posts/9-01-2015-digest13.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Hi folks! 
+
+{DESC}
+
+* \[Scala\] [Abstract types in Scala](https://www.hakkalabs.co/articles/scala-bay-tech-talk-abstract-types#)
+
+* \[Scala\] [Scala puzzlers](http://scalapuzzlers.com/)
+
+* \[Scala\] [Why Spark Is the Next Top (Compute) Model](http://www.infoq.com/presentations/spark-scala-mapreduce-java)
```
Comment (pjazdzewski1990) [inline _posts/9-01-2015-digest13.md]: :+1:

File: _posts/9-01-2015-digest13.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+* \[Frontend\] [On the road to 2015](https://medium.com/@addyosmani/javascript-application-architecture-on-the-road-to-2015-d8125811101b)
+ 
+* \[Game Development\] [From Never Doing Game Code or Game Art... To A Full Game](http://www.gamedev.net/page/resources/_/business/breaking-into-the-industry/from-never-doing-game-code-or-game-art-to-a-full-game-in-unreal-engine-4-r3937)
+
+* \[Working\] [Devs should write](http://www.shubhro.com/2014/12/27/software-engineers-should-write/)
+
+* \[Startups\] [Building an Incredible Founding Team](https://equityzen.com/blog/building-an-incredible-founding-team/)
+
+* \[Biotechnology\] [The IT Wizard](http://nautil.us/issue/7/waste/nature-the-it-wizard)
```
Comment (serafinmaciej) [inline _posts/9-01-2015-digest13.md]: :+1:

File: _posts/9-01-2015-digest13.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+- Scala
+---
+
+Hi folks! 
+
+Welcome to the first digest in 2015. We are happy to be with you for the second year. Hope you will learn and get inspired by the material we prepared for you. See you next week.
+
+* \[Scala\] [Scala Puzzlers](http://scalapuzzlers.com/)
+
+Scala puzzlers is a initiative that brings together traps, pitfalls and corner cases of Scala. Here you can test how good is your understanding of language ... and get surprised a few times. Better to learn it here while having fun, rather than when debuging a production system.
```
Comment (rtkaczyk) [inline _posts/9-01-2015-digest13.md]: how good is your understanding of language -> how good your understanding of the language is

File: _posts/9-01-2015-digest13.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+Welcome to the first digest in 2015. We are happy to be with you for the second year. Hope you will learn and get inspired by the material we prepared for you. See you next week.
+
+* \[Scala\] [Scala Puzzlers](http://scalapuzzlers.com/)
+
+Scala puzzlers is a initiative that brings together traps, pitfalls and corner cases of Scala. Here you can test how good is your understanding of language ... and get surprised a few times. Better to learn it here while having fun, rather than when debuging a production system. 
+
+* \[Scala\] [Why Spark Is the Next Top (Compute) Model](http://www.infoq.com/presentations/spark-scala-mapreduce-java)
+
+Apache Spark is said to be the next big thing of big data and a successor to Hadoop. In this video Dean Wampler argues why this Scala based framework is superior to Hadoop and how can it be used to face modern data challenges like streaming, storing data and distributing computation.
```
Comment (rtkaczyk) [inline _posts/9-01-2015-digest13.md]: can it -> it can

File: _posts/9-01-2015-digest13.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+Scala puzzlers is a initiative that brings together traps, pitfalls and corner cases of Scala. Here you can test how good is your understanding of language ... and get surprised a few times. Better to learn it here while having fun, rather than when debuging a production system. 
+
+* \[Scala\] [Why Spark Is the Next Top (Compute) Model](http://www.infoq.com/presentations/spark-scala-mapreduce-java)
+
+Apache Spark is said to be the next big thing of big data and a successor to Hadoop. In this video Dean Wampler argues why this Scala based framework is superior to Hadoop and how can it be used to face modern data challenges like streaming, storing data and distributing computation. 
+
+* \[Frontend\] [On the road to 2015](https://medium.com/@addyosmani/javascript-application-architecture-on-the-road-to-2015-d8125811101b)
+
+In his article Addy Osmani described what changed in frontend programming in 2014 and what might become a huge in the upcoming year.
```
Comment (rtkaczyk) [inline _posts/9-01-2015-digest13.md]: a huge what?

File: _posts/9-01-2015-digest13.md (hunk 5 above)
Comment (rtkaczyk) [inline _posts/9-01-2015-digest13.md]: a initiative -> an initiative

File: _posts/9-01-2015-digest13.md (hunk 7 above)
Comment (pjazdzewski1990) [inline _posts/9-01-2015-digest13.md]: ... become huge ...
